### PR TITLE
Okwme/e2e updates

### PR DESCRIPTION
**Pull request by okwme**
updated the e2e to address the failing upgrade proposal test that was removed in #1625

### CODE REVIEW

**Comment by yaruwangway** [comment]
Hi @okwme , does this pr contain refactor ? if yes, can we remove the refactor part ? and leave refactor todo when all open e2e pr merged .. otherwise, too much conflicts.

**Comment by okwme** [comment]
@yaruwangway good idea!

**Comment by okwme** [comment]
marked as draft until then @yaruwangway

**Comment by yaruwangway** [comment]
thanks !

File: tests/e2e/e2e_util_test.go [hunk 1]
```diff
@@ -310,64 +246,104 @@ func (s *IntegrationTestSuite) execGovSubmitProposal(c *chain, valIdx int, endpo
...
+
+	exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
+		Context:      ctx,
+		AttachStdout: true,
+		AttachStderr: true,
+		Container:    s.valResources[c.id][valIdx].Container.ID,
+		User:         "root",
+		Cmd: append(gaiaCommand,
+			fmt.Sprintf("--%s=%s", flags.FlagFrom, from),
+			fmt.Sprintf("--%s=%d", flags.FlagSequence, sequence),
```
Comment (yaruwangway) [inline tests/e2e/e2e_util_test.go:282]: sequence+1 ?

Comment (okwme) [reply tests/e2e/e2e_util_test.go:282]: nope it's sequence, i thought the same thing at first

**Pull request closed without merging**